src/merkle/merkle-objects.ts

```diff
@@ -10,8 +11,13 @@ import { IpfsService } from '../services/ipfs-service.js'
 import { BloomFilter } from 'bloom-filters'
 import { StreamID } from '@ceramicnetwork/streamid'
 
+const packageJson = JSON.parse(
+  fs.readFileSync(new URL('../../package.json', import.meta.url), 'utf8')
+)
+
 const BLOOM_FILTER_TYPE = 'jsnpm_bloom-filters'
 const BLOOM_FILTER_FALSE_POSITIVE_RATE = 0.0001
+const BLOOM_FILTER_VERSION = packageJson['dependencies']['bloom-filters']
```

Comment:
unfortunately this approach will only tell us the version that is specified in the package.json, not what was actually installed by npm, which could be different.  Given that the bloom-filters package literally had a breaking change in a patch version release (which is what started all this), we need to record the *actual* version being used.  I don't actually know if there's an easy way to figure that out at runtime in javascript, maybe @ukstv has an idea

Reply:
AHHH i see i would have hoped that they respected semver hahaha

Reply:
A similar to solution to what I have here is using `node_modules/bloom-fiters/package.json` instead of our `package.json` dependencies. This solution will provide the exact version installed. I am not sure if this is the best solution so i'll push the code and see what @ukstv says.

Reply:
I am okay with that. As we discussed with @stephhuynh18 , better way would be to have our own bloom-filter (or XOR filter) implementation, to reduce potential risks from external dependency mismanagement. But, it increases our scope, so I am all right with the current @stephhuynh18 approach.